src/operations/client_bulk_write/common.ts

```diff
@@ -181,6 +182,55 @@ export interface ClientBulkWriteResult {
   deleteResults?: Map<number, ClientDeleteResult>;
 }
 
+export interface ClientBulkWriteError {
+  code: number;
+  message: string;
+}
+
+/**
+ * An error indicating that an error occurred when executing the bulk write.
+ *
+ * @public
+ * @category Error
+ */
+export class MongoClientBulkWriteError extends MongoServerError {
+  /**
```

Comment:
Thoughts on moving this to `errors.ts` so all of our bulk write eerrors live together?  Or moving the other bulk write error from errors.ts into this file?  Either way.

Reply:
Moved to error.ts